internal/gke/gke.go

```diff
@@ -451,8 +439,8 @@ func ensureReplicaSetAutoscaler(ctx context.Context, cluster *ClusterInfo, logge
 	if err := autoJSONTmpl.Execute(&b, struct {
 		Name             string
 		Namespace        string
-		AppName          string
-		DeploymentID     string
+		AppNameLabel     string
```

Comment:
Are these labels somehow interpreted by GKE automatically? I don't see how we  use them anywhere in our code.